src/Game/UI/uiPauseMenuDataMgr.h

```diff
@@ -370,6 +370,12 @@ class PauseMenuDataMgr {
     // FIXME: types
     bool useItemFromRecipeAndSave(void* unk, int multiplier, PouchItem* item);
 
+    void grabbedItemStuff(PouchItem* item);
+
+    static void dropItemStuff(const char* name, sead::Heap* heap, int w2_0, bool a4, s64 a5,
+                              bool a6, float a7, float a8);
+    void updateInventoryCategories(const sead::OffsetList<PouchItem>& list);
```

Comment:
I think this already exists (`updateInventoryInfo`) -- it might be worth adding a comment with the address of this function to avoid any further confusion (sorry for not doing that earlier!)